src/scenes/AcceleratorRouter/Documents/DocumentView/DocumentVariablesTab.tsx

```diff
@@ -162,10 +161,12 @@ const DocumentVariablesTab = ({ document: doc, setSelectedTab }: DocumentVariabl
   );
 
   const onUpdate = () => {
-    dispatch(requestListVariables(doc.variableManifestId));
+    dispatch(requestListAllVariables());
     dispatch(requestListVariablesValues({ projectId: doc.projectId }));
   };
 
+  const onSearch = (str: string) => setSearchValue(str);
```

Comment:
I noticed that this does not actually do anything, the search value isn't checked anywhere. I will fix in a subsequent PR

Reply:
https://github.com/terraware/terraware-web/pull/2867